### ISSUE TITLE
Why the web app greyed out within few minutes of inactivity

### ISSUE THREAD

**Issue by noyara3** [royfrancis/pophelperShiny#7]
Within only 1-2 min. if I leave the shiny app without any click it will be greyed out. Which force me to reload the data and start again. 

Can I increase the time of waiting for me before going to sleep?

**Comment by noyara3** [comment]
Btw
I mean this webapp:
https://pophelpershiny.serve.scilifelab.se/

**Comment by royfrancis** [comment]
Yes, I understand it's annoying. Unfortunately, this is out of my control. It is to do with the hosting service. I have asked them to fix this several times. You can try to run the app locally on your computer using docker as described [here](royfrancis.github.io/pophelperShiny/). But you need to first install docker.
